app/assets/javascripts/components/student/pages/lessons_page.es6.jsx

```diff
@@ -1,12 +1,35 @@
 class LessonsPage extends React.Component {
 
+  static get propTypes() {
+    return {
+      studentId: React.PropTypes.number.isRequired,
+    }
```

Comment:
i think it should be a comma here for consistency because all other `static get propTypes` in our codebase have commas, even if it's just one line.